charts/seed-operatingsystemconfig/original/templates/containerd/containerd-config-generator.service

```diff
@@ -0,0 +1,15 @@
+{{- define "containerd-config-generator" -}}
+- name: containerd-config-generator.service
+  command: start
+  enable: true
+  content: |
+    [Unit]
+    Description=Containerd config generator
+    [Install]
+    WantedBy=multi-user.target
+    [Service]
+    Type=oneshot
+    RemainAfterExit=yes
+    ExecStart=/opt/bin/init-containerd
```

Comment:
This just calls `/opt/bin/init-containerd` maybe a better name would be `containerd-initializer.service` ?

Reply:
Thanks for the suggestion, though I think containerd-config-generator.service is more precise at the moment as it really just starts the config generation. 
If there would be more initialization going on I would agree.
Would that be fine with you @zanetworker ?

Reply:
I am fine, it was just a suggestion based on the fact that context of the script is not clear  from the binary executed, but again fine by me :)

Reply:
I changed my mind, I ll rename it as you suggested. There might be more changes included here in the future that goes beyond generating the config file.